.github/workflows/pull-request.yml

```diff
@@ -4,7 +4,6 @@ name: Workflow - Pull Request
 on:
   workflow_dispatch:
   pull_request:
-    branches: [main]
```

Comment:
should we undo this?

Reply:
yes will do, was here to test out that the things are working fine

Reply:
pending

Reply:
done